packages/components/input/src/input.vue

```diff
@@ -2,18 +2,18 @@
   <div
     v-show="type !== 'hidden'"
     :class="[
-      type === 'textarea' ? 'el-textarea' : 'el-input',
-      inputSize ? 'el-input--' + inputSize : '',
+      type === 'textarea' ? nsTextarea.b() : nsInput.b(),
+      nsInput.m(inputSize),
       {
-        'is-disabled': inputDisabled,
-        'is-exceed': inputExceed,
-        'el-input-group': $slots.prepend || $slots.append,
-        'el-input-group--append': $slots.append,
-        'el-input-group--prepend': $slots.prepend,
-        'el-input--prefix': $slots.prefix || prefixIcon,
-        'el-input--suffix':
+        [nsInput.is('disabled')]: inputDisabled,
+        [nsInput.is('exceed')]: inputExceed,
+        [nsInput.b('group')]: $slots.prepend || $slots.append,
+        [nsInput.b('group--append')]: $slots.append,
```

Comment:
Maybe we can add a new namespace `input-group` and use
```suggestion
        [nsInputGroup.m('append')]: $slots.append,
```

Reply:
Wait. We can add a function called `bm`. And then we needn't add a new namespace.

Reply:
I will add it in a new PR.